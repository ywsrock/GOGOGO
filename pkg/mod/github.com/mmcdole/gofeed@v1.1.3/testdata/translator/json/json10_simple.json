{
  "version": "1.0",
  "items": [
    {
      "content_text": "content_text",
      "banner_image": "https://sample-json-feed.com/banner_image.png"
    }
  ]
}
