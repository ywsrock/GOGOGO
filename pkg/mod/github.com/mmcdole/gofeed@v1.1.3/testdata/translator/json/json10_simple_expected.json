{
	"feedVersion": "1.0",
	"feedType": "json",
	"items": [
		{
			"content": "content_text",
			"image": {
				"url": "https://sample-json-feed.com/banner_image.png"
			}
		}
	]
}